include/ck/tensor_operation/gpu/device/device_conv2d_fwd_xdl_c_shuffle_bias_activation_add_nhwc_kyxc_nhwk.hpp

```diff
@@ -576,6 +573,8 @@ struct
             c0_grid_desc_m_n_    = descs[I3];
             c1_grid_desc_m_n_    = descs[I4];
 
+            block_2_ctile_map_ = Block2CTileMap{c_grid_desc_m_n_};
```

Comment:
The bug probably owes to the same as the fix in #258, where C tile map is prematurely initialized before C descriptor is initialized. As with what you did in this line.

Now I have no idea how this test could even pass after I messed this up by accident. Anyone?

Reply:
conv ctest is initializing everything to 0.

This PR is trying to fix it
https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/259

Reply:
@asroy  We do not have any other tests for conv fwd + bias + activation + add other than just the example.